deps/3rdparty/libdispatch/build/libdispatch.vcxproj

```diff
@@ -125,8 +109,8 @@
       <GenerateDebugInformation>true</GenerateDebugInformation>
       <LargeAddressAware>true</LargeAddressAware>
       <ImageHasSafeExceptionHandlers>true</ImageHasSafeExceptionHandlers>
-      <AdditionalDependencies>libobjc2.lib;mincore.lib;%(AdditionalDependencies)</AdditionalDependencies>
-      <AdditionalLibraryDirectories>"$(WINOBJC_SDK_ROOT)\build\$(Platform)\$(Configuration)\Universal Windows"</AdditionalLibraryDirectories>
+      <AdditionalDependencies>libobjc2.lib;%(AdditionalDependencies)</AdditionalDependencies>
+      <AdditionalLibraryDirectories>"$(WINOBJC_SDK_ROOT)\tools\deps\prebuilt\Universal Windows\x86\Debug\"</AdditionalLibraryDirectories>
```

Comment:
$(Platform) and $(Configuration) don't work anymore?

Reply:
I believe this is because its x86 not Win32. You should use `$(TargetPlatform)` instead. Take a look at some of the other msbuild files in the tree for this.

---
In reply to: [141207703](https://github.com/Microsoft/WinObjC/pull/2799#discussion_r141207703) [](ancestors = 141207703)

Reply:
If you want to super extra double awesome, you could also factor this out of the per configuration blocks into a global ItemDefinitionGroup

---
In reply to: [141459171](https://github.com/Microsoft/WinObjC/pull/2799#discussion_r141459171) [](ancestors = 141459171,141207703)